wattwatchers_rest_api_v3_long_energy.ipynb

```diff
@@ -381,7 +373,16 @@
         "    return results\n",
         "\n",
         "  # Helper method to generate a dictionary with an id, data and (optionally) device metadata\n",
-        "  def device_le_dict_for(device_id, data, metadata):\n",
+        "  def device_le_dict_for(device_id, timezone, data, metadata):\n",
+        "    if INCLUDE_DATE_AND_TIME:\n",
+        "      tz = pytz.timezone(timezone)\n",
+        "      def _include_date_and_time(interval: dict) -> dict:\n",
+        "        datetime = dt.datetime.fromtimestamp(interval['timestamp'], tz)\n",
+        "        interval['date'] = datetime.strftime('%m/%d/%Y')\n",
```

Comment:
Assuming the MM/DD format is what Excel expects?

Reply:
Yeah, tested importing in Excel (after some Googling for supported date formats). This worked, so left it at that.